app/assets/stylesheets/application.scss

```diff
@@ -61,3 +61,30 @@
     padding: 0px 10px;
   }
 }
+
+.authorizations.new {
+    #application-body {
```

Comment:
You might consider adding the `ox-card` style to the view, that way it'll match the other "card type" views and can use the built-in styles https://github.com/openstax/accounts/blob/master/app/assets/stylesheets/controls/_card.scss

Will probably still have to style the buttons a bit though

Reply:
The view seems to be owned by the doorkeeper gem, as you suggested, so unless I make a local view (and I suspect that's beyond my skills), I can't add a class. If `ox-card` were defined as a placeholder type, though, I could use that.

Reply:
ah gotcha.  Yep this makes sense then